src/Provider/ConfigServiceProvider.php

```diff
@@ -147,13 +147,6 @@ private function loadConfig()
             ));
         }
 
-        if ($this->filename->getExtension() != 'yml') {
```

Comment:
Hi, thanks for the PR.

I think it would be better to add the yaml extension to the if statement.

`if ($this->filename->getExtension() != 'yml' || $this->filename->getExtension() != 'yaml') {`